src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -158,6 +178,7 @@ export const VolunteerFormCalendar = ({
   return (
     <>
       <Wrapper>
+        <Input hidden name="calendarId" control={control} />
         <Controller
           name="contentContainerId"
           render={({ name, onChange, value }) => (
```

Comment:
when editing, this is not allowed to be changed thus should not be visible

<img width="385" alt="image" src="https://user-images.githubusercontent.com/1942953/199038426-e8daea54-4a06-48ce-9235-0604e203de23.png">

Reply:
and if an event is edited, the succes messge should be something like "erfolgreich aktualisiert" instead of "erstellt"

<img width="296" alt="image" src="https://user-images.githubusercontent.com/1942953/199042820-98c1c219-d2ac-4f8e-9bca-8f6cd6e1244a.png">

Reply:
✔️ 

<img width="287" alt="image" src="https://user-images.githubusercontent.com/1942953/199043677-517fb3fc-21c6-4fb4-886c-242fe74a873b.png">

Reply:
after editing the details should update automatically. i have changed the title but after navigation the title stays the old one

Reply:
i have found a solution with `useFocusEffect()` that we already use in other places

Reply:
01b627a